casbin_redis_watcher/watcher.py

```diff
@@ -66,45 +66,67 @@ def update(self):
         def func():
             with self.mutex:
                 msg = MSG("Update", self.options.local_ID, "", "", "")
-                return self.pub_client.publish(self.options.channel, msg.marshal_binary())
+                return self.pub_client.publish(
+                    self.options.channel, msg.marshal_binary()
+                )
```

Comment:
@Nekotoxin  Please modify these lines into one line, the same below.

Reply:
changed